reference/functions/packagesmatching.markdown

```diff
@@ -39,7 +39,12 @@ this:
 The following code extracts just the package names, then looks for
 some desired packages, and finally reports if they are installed.
 
-**IMPORTANT:** Please note that you need to provide `package_inventory` attribute in `body common control` in order to be able to use this function. Also depending on the value(s) of `package_inventory` only packages from selected package modules will be returned. For more information about `package_inventory` please read [`package_inventory`][Components#package_inventory] section.
+**IMPORTANT:** Querying operates on either standard package modules or legacy package methods.
+When `package_inventory` in `body common control` is configured, CFEngine will record the packages installed and the package updates available for the configured package modules.
+In the [Masterfiles Policy Framework][Masterfiles Policy Framework] `package_inventory` will be [https://github.com/cfengine/masterfiles/blob/master/promises.cf.in#L92][configured] to the default for the hosts platform.
```

Comment:
```suggestion
In the [Masterfiles Policy Framework][Masterfiles Policy Framework] `package_inventory` will be [configured](https://github.com/cfengine/masterfiles/blob/3dc1f629544b24261975ecf86e02554d4daf346e/promises.cf.in#L92) to the default for the hosts platform.
```
The link format was wrong for markdown and probably better to use a permalink rather than line based on moving branch.